src/tests/data/xof/shake.vec

```diff
@@ -0,0 +1,539 @@
+# These vectors are selected from the NIST CAVS file for SHAKE
+#  Generated on Thu Jan 28 14:46:49 2016
+
+[SHAKE-128]
```

Comment:
Note to self: the Keccak permutation provides a `bmi2` implementation. Probably, we want to insert

```
#test cpuid bmi2
```

in that test vector file to exercise the alternative provider.